src/bootloader/bootloader.c

```diff
@@ -358,33 +358,44 @@ static void _render_progress(float progress)
 
 static void _render_hash(const char* title, const uint8_t* hash)
 {
-    uint8_t seconds = 10;
-    char message[16];
+    // If you change this, check the timer_buf size below.
+    const uint8_t seconds = 10; // how many seconds to show screen
+    const UG_S16 title_margin = 7; // Margin between title and hash
+    const UG_FONT* f_mono = &font_monogram_5X9; // monospaced font
+    const UG_FONT* f_regular = &font_font_a_9X9; // regular font
+
+    // Convert hash to ascii hex
     char hash_hex[2 * SHA256_DIGEST_LENGTH + 1];
     util_uint8_to_hex(hash, SHA256_DIGEST_LENGTH, hash_hex);
-    char scratch = 0;
+
+    // Buffer for timer. 3 bytes would be enough to hold the string up to "9s", but we do a bit more
+    // just in case there the number of seconds changes.
+    char timer_buf[4];
+    UG_S16 timer_len = 0; // Store the length of the timer string
```

Comment:
> Store the length of the timer string

it sounds like `len(timer_buf)`, as in the number of characters, but it's actually a string width on the screen in pixels. Maybe adding the unit to the comment would make it less confusing imho.

Reply:
Good idea. Done and renamed the var to _width.

Reply:
thanks!